xt/66-cucumber/13-cash/step_definitions/basic_steps.pl

```diff
@@ -189,8 +189,8 @@ sub _create_recon {
     ok(1);
 };
 
-Then qr/(the (first |second )?reconciliation) can( also| not|'t)? be submitted/ => sub {
-    my $negate     = $2 and ($2 eq ' not' or $2 eq q{'t});
+Then qr/(the (?:first |second )?reconciliation) can( also| not|'t)? be submitted/ => sub {
+    my $negate     = ($2 and ($2 eq ' not' or $2 eq q{'t}));
```

Comment:
I think that 193 should read `my $negate = ($3 and ($3 eq ' not' or $3 eq q{'t}));`. The first parenthesis gets the whole sentence in $1